libs/common/src/config/crypto.config.ts

```diff
@@ -0,0 +1,20 @@
+import { z } from 'zod';
+import dotenv from 'dotenv';
+
+dotenv.config();
+
+const BYTE_GENERATOR_PROVIDERS = ['cryptographically-strong'] as const;
+const byteGeneratorProviders = z.enum(BYTE_GENERATOR_PROVIDERS).default('cryptographically-strong');
+type ByteGeneratorProviders = z.infer<typeof byteGeneratorProviders>;
+
+const HASH_STRATEGY_PROVIDERS = ['pbkdf2-sha512'] as const;
+const hashStrategyProviders = z.enum(HASH_STRATEGY_PROVIDERS).default('pbkdf2-sha512');
+type HashStrategyProviders = z.infer<typeof hashStrategyProviders>;
```

Comment:
We actually don't need this now -- im going to change it so zod verifies string and then we handle a bad string in our switch case